README.md

```diff
@@ -16,3 +17,16 @@ page](https://docs.github.com/en/get-started/writing-on-github/working-with-adva
 might help with the notation for mathematical expressions.
 
 $f(n)\in o(g(n)) \iff \forall c>0, \exists n_0, \forall n\ge n_0: f(n) < c g(n)$
+
+The definition for big-O is:
+
+$f(n)\in O(g(n)) \iff \exists c>0, \exists n_0, \forall n\ge n_0: f(n) \leq c g(n)$
+
+Considering the same $f(n) and  g(n)$. If $f(n) \in o(g(n))$, we know that any value of c greater than 0 (and a n_0) satisfies $ \forall n\ge n_0: f(n) < c g(n) $. 
+$\forall c > 0 \implies \exists c >0 $, so based on the big-O definition we can say that $f(n) \in O(g(n))$.
```

Comment:
Your markup doesn't render correctly here. Also please be explicit about what part of the definition implies what.